### ISSUE TITLE
UX: Sharing toolbar between editables

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5-design#100]
## What are the pros and cons of sharing a toolbar?

* Should there be an indicator in the toolbar which would identify which editable is being edited at the moment?
* Should there be a some sort of navigation in the toolbar to switch between editables?
  * Especially in the context of a11y.

----

![image](https://cloud.githubusercontent.com/assets/1099479/12417863/6cff8e7c-beab-11e5-8990-501999f51e48.png)

**Comment by SteveTheTechie** [comment]
My 2 cents...  Don't add a lot of "fat" to the toolbar...  Keep it "lean".  Look for other, more creative ways to accomplish the same thing.   In this case, you might define and allow redefinition of a CSS style that would essentially "highlight" the text area being edited.  For example, maybe a heavy colored border is applied to the text area being edited.  (or a background color)

**Comment by fredck** [comment]
I agree that the toolbar should be kept essential. If any a11y feature is required for navigation, this could be made available under a keystroke.

**Comment by oleq** [comment]
@SteveTheTechie, @fredck Makes sense to me. Just poking around to see if there's another, better solution hidden somewhere.

**Comment by koleary** [comment]
Yes, but preferably in a floating toolbar at the position of the _cursor_.
